android-opencv/build.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+# ==================================================================================================
+# Copyright (c) 2018, Jairus Martin.
+# Copyright 2018 by Rodrigo Gomes <rodgomesc@gmail.com>
+# Distributed under the terms of the GPL v3 License.
+# The full license is in the file LICENSE, distributed with this software.
+# Created on Nov 22, 2018
+# ==================================================================================================
+
+# need Android SDK Tools to version 25.2.5
+# Details: https://github.com/opencv/opencv/issues/8460
+
+export ARCHS=("x86_64 x86 arm arm64")
+export NDK="$HOME/Android/Sdk/ndk-bundle"
+export ANDROID_SDK_ROOT="$HOME/Android/Sdk"
+
+for ARCH in $ARCHS
+do
+
+    CONFIGURE_FLAGS=""
+    if [ "$ARCH" == "arm" ]; then
+        export TARGET_HOST="arm-linux-androideabi"
+        export TARGET_ABI="armeabi-v7a"
+    elif [ "$ARCH" == "arm64" ]; then
+        export TARGET_HOST="aarch64-linux-android"
+        export TARGET_ABI="arm64-v8a"
+        
+    elif [ "$ARCH" == "x86" ]; then
+        export TARGET_HOST="i686-linux-android"
+        export TARGET_ABI="x86"
+    elif [ "$ARCH" == "x86_64" ]; then
+        export TARGET_HOST="x86_64-linux-android"
+        export TARGET_ABI="x86_64"
+    fi
+
+    export ANDROID_TOOLCHAIN="$NDK/standalone/$ARCH"
+    export APP_ROOT="$PREFIX/android/$ARCH"
+    export PATH="$PATH:$ANDROID_TOOLCHAIN/bin"
+    export AR="$TARGET_HOST-ar"
+    #export AS="$TARGET_HOST-clang"
+    export CC="$TARGET_HOST-clang"
+    export CXX="$TARGET_HOST-clang++"
+    export LD="$TARGET_HOST-ld"
+    
+    if [ "${TARGET_ABI}" = "armeabi" ]; then
+        API_LEVEL=19
+    else
+        API_LEVEL=21
+    fi
+
+    rm -Rf build || true
+    mkdir build
+    cd build       
+	cmake .. -DCMAKE_BUILD_WITH_INSTALL_RPATH=ON \
+	      -DCMAKE_TOOLCHAIN_FILE="$NDK/build/cmake/android.toolchain.cmake" \
+              -DANDROID_ARM_NEON=ON \
```

Comment:
This flag should be set based on the arch

Reply:
ok go fix this

Reply:
-DANDROID_ARM_NEON=ON is arm/arm64 only and will not work on x86*

Reply:
i think opencv automatically does it for us 

https://github.com/opencv/opencv/blob/c6daa4aa1661fed7c7a4de5a10a2f8953bf2757e/CMakeLists.txt#L328

Reply:
Good call. thanks

Reply:
Good call. thanks